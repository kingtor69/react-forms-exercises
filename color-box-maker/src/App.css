.App {
	background-color: #7d7d7d;
	color: white;
	min-width: 100vw;
	min-height: 100vw;
	text-align: center;
}

.form {
	margin: 0 5vw;
	width: 80vw;
	padding: 5vh 5vw;
	border: 2px solid black;
	border-radius: 10px;
}

.form-group {
	margin: 5px;
}

input,
button {
	border-radius: 5px;
	padding: 3px;
	margin: 0 5px;
}

.Box {
	margin: 4vh 5vw;
	border: 1px solid black;
}
