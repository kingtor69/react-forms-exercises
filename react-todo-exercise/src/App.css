.form {
	margin: 0 5vw;
	width: 80vw;
	padding: 5vh 5vw;
	border: 2px solid black;
	border-radius: 10px;
}

.form-group {
	margin: 5px;
}

input,
button {
	border-radius: 5px;
	padding: 3px;
	margin: 0 5px;
}

ul {
	list-style: none;
}

.deleteTodoButton {
	background-color: #bbb;
	color: #000;
	padding: 0 3px;
	margin: 0 6px;
}
