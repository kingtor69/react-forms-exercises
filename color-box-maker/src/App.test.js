import { render, asFragment } from '@testing-library/react';
import App from './App';

it('renders witout ceashing', () => {
  render(<App />);
});

it('renders snapshot', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
})