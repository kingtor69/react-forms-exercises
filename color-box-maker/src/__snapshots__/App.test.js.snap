// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="NewBoxList"
    >
      <div
        class="NewBoxForm"
      >
        this is a form to make a new box
      </div>
      <div
        class="Box"
        data-testid="box0"
        style="background-color: blue; width: 150px; height: 90px;"
      />
    </div>
  </div>
</DocumentFragment>
`;
